docs/pages/elevation.js

```diff
@@ -1,10 +1,34 @@
 // @flow strict
 import { type Node } from 'react';
-import { Box, Flex, Text } from 'gestalt';
+import { Box, ColorSchemeProvider, Flex, Text } from 'gestalt';
 import MainSection from '../components/MainSection.js';
 import PageHeader from '../components/PageHeader.js';
 import Page from '../components/Page.js';
 
+// eslint-disable-next-line react/prop-types
+function ColorCard({ description, colorScheme, id, boxProps }) {
+  return (
+    <Flex gap={4} direction="column">
+      <ColorSchemeProvider colorScheme={colorScheme} id={id}>
+        <Box padding={6} color="default">
+          <Box
+            alignItems="center"
+            justifyContent="center"
+            display="flex"
+            width={300}
+            height={300}
+            rounding={3}
+            {...boxProps}
```

Comment:
Let's avoid spreading props if at all possible. This will make any future codemods impossible.